pythonforandroid/recipes/python3/__init__.py

```diff
@@ -71,17 +71,22 @@ class Python3Recipe(TargetPythonRecipe):
         # Python 3.8.1 & 3.9.X
         ('patches/py3.8.1.patch', version_starts_with("3.8")),
         ('patches/py3.8.1.patch', version_starts_with("3.9")),
-        ('patches/py3.8.1.patch', version_starts_with("3.10"))
+        ('patches/py3.8.1.patch', version_starts_with("3.10")),
     ]
 
     if shutil.which('lld') is not None:
-        patches = patches + [
+        patches += [
             ("patches/py3.7.1_fix_cortex_a8.patch", version_starts_with("3.7")),
             ("patches/py3.8.1_fix_cortex_a8.patch", version_starts_with("3.8")),
             ("patches/py3.8.1_fix_cortex_a8.patch", version_starts_with("3.9")),
-            ("patches/py3.8.1_fix_cortex_a8.patch", version_starts_with("3.10"))
+            ("patches/py3.8.1_fix_cortex_a8.patch", version_starts_with("3.10")),
+            ("patches/py3.11.4_fix_cortex_a8.patch", version_starts_with("3.11"))
```

Comment:
`patches/py3.8.1_fix_cortex_a8.patch` applies on `3.11`, therefore a new `patches/py3.11.4_fix_cortex_a8.patch` is not needed.

Can you please just change: (as we did for 3.10)
```python
("patches/py3.11.4_fix_cortex_a8.patch", version_starts_with("3.11"))
```
to
```python
("patches/py3.8.1_fix_cortex_a8.patch", version_starts_with("3.11"))
```

and remove the `patches/py3.11.4_fix_cortex_a8.patch` file?